dist/cargo_test.sh

```diff
@@ -7,12 +7,14 @@ cargo_test_flags["cincinnati"]="--features test-net"
 cargo_test_flags["commons"]=""
 cargo_test_flags["graph-builder"]="--features test-net"
 cargo_test_flags["policy-engine"]=""
+cargo_test_flags["prometheus-query"]=""
```

Comment:
We can have test-net by default

```suggestion
cargo_test_flags["prometheus-query"]="--features test-net"
```

Reply:
Note: I'm not sure if the `test-net` feature gates any tests right now. I also don't have an opinion of whether `test-net-private` should require `test-net` to be set as well. Please make a decision for this and possibly drop `test-net` from the crate if it's unused.

Reply:
Removed it - the only remaining prometheus_query test in `prometheus_query/src/v1/queries/mod.rs` doesn't need any special flags